.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+### Describe your changes
+
+
+### Related issue number or link
+
+
```

Comment:
This will automatically link related issues to the PR, and enable [keyword-based auto closing of related issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue):
(For information: The auto-closing feature will be effective only for the PRs that target `main` branch.)

```suggestion

Resolves #ISSUE-NUMBER

```

@seokho-son PTAL!

Reply:
Hi @jihoon-seo 
Thank you !

I've been considered about adding an example problem number too. But I decided not to.
Because not every PR is based on an Issue.

How about this alternative ?